code/modules/client/preferences.dm

```diff
@@ -1628,8 +1628,8 @@ NOTE:  The change will take effect AFTER any current recruiting periods."}
 	character.backbag = backbag
 
 	//Debugging report to track down a bug, which randomly assigned the plural gender to people.
-	if(character.gender in list(PLURAL, NEUTER))
-		if(isliving(character)) //Ghosts get neuter by default
+	if(character.gender in list(PLURAL, NEUTER) && chosen_species != "Mushroom")
```

Comment:
Snowflake since unatomic, i have a perfect tag for this.

Reply:
guess i'll do it in a seperate PR at some point, whatever